agenda/data/src/main/java/com/gkp/agenda/data/OfflineAgendaRepository.kt

```diff
@@ -51,6 +50,55 @@ class OfflineAgendaRepository(
         }.launchIn(scope)
     }
 
+    override fun updateAgendaItem(agendaItem: AgendaItem) {
+        when(agendaItem){
+            is AgendaItem.Event -> {
+                networkApiCall {
+                    val eventBody = agendaItem.toEventBody()
+                    val eventBodyPart = buildEventBodyPart(eventBody)
+                    val photosPart = buildPhotosPart(agendaItem.photos)
+                    taskyRetrofitApi.updateEvent(
+                        eventBodyPart = eventBodyPart,
+                        photosPart = photosPart
+                    )
+                }
+
+            }
```

Comment:
Could it also make sense to update your local database here?

Reply:
I will implement the caching in the next PR